handler.js

```diff
@@ -15,12 +20,12 @@ const connectToDatabase = require('./dataLayer');
 
 const server = require('apollo-server-lambda');
 
-exports.graphqlHandler = function graphqlHandler(event, context, callback) {
+export async function graphqlHandler(event, context, callback) {
```

Comment:
I do not know why but if I run this locally I get:

```sh
[ 'Error: Serverless-offline: handler for \'graphql\' is not a function',
```

It works fine when I change it to:

```js
exports.graphqlHandler = async function (event, context, callback) {
```

Reply:
I will revert